### PR TITLE
Add an example grafana dashboard

### PR DESCRIPTION
**Pull request by avi-biton**
Add an example grafana dashboard

### CODE REVIEW

**Comment by avi-biton** [comment]
@gbenhaim @amisstea This is a PR for adding an example dashboard so we can demonstrate to other teams the correct way to add dashboards

PTAL :)

File: grafana/dashboards/o11y-dashboard.json
Comment (amisstea) [inline grafana/dashboards/o11y-dashboard.json:1]: Have you considered adding some basic linting for dashboards in the CI? There's a relatively new linter that might be useful: https://github.com/grafana/dashboard-linter

Comment (avi-biton) [reply grafana/dashboards/o11y-dashboard.json:1]: Should we open a ticket for it? or should we add it to this one?

Comment (avi-biton) [reply grafana/dashboards/o11y-dashboard.json:1]: Looking at it and testing it, its in early phases and focused exclusively on dashboards that use a Prometheus data source.

We remove the datasource from our json files to get the default datasource.
In my tests it fails with error: `Dashboard 'o11y' does not have a templated data source`.
So, I think it is not suited for us.
WDYT?

Comment (amisstea) [reply grafana/dashboards/o11y-dashboard.json:1]: It seems we might be able to [exclude](https://github.com/grafana/dashboard-linter/blob/main/docs/index.md#exclusions-and-warnings) that [rule](https://github.com/grafana/dashboard-linter/blob/main/docs/rules/template-datasource-rule.md).

**Pull request merged**